### ISSUE TITLE
Accessibility Issue: "Perceivability"

### ISSUE THREAD

**Issue by larrykind** [globaleaks/GlobaLeaks#3373]
### What version of GlobaLeaks are you using?

4.10.18 - development environment

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### Describe the issue

Hello everyone, 
this is a first summarization on our accessibility report. This summarization collects all accessibility issue about the Perceivability, tha capabilty of the user with reduced mobility to use Globaleaks with screen readers.

| **Index** | **Issue Description**                                                                                                                                                                                          | **Ref. Check UNI CEI EN 301549:2021** | **conformity level** | **status** |
|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|----------------------|----------------------|
| 1         | *The logo shows a non appropriate alternative text alt="project logo" independently of the project's name                                                                                                      | C.9.1.1.1                             | A                    | FIXED |
| 2         | *The headers and the logo are allocated inside elements with role button: --> delete role button.                                                                                                              | C.9.2.4.6                             | AA                   | FIXED |
| 3         | *The input filed near the button "Login" is not associated to any label : ---> add . As a content of the  element,  the text "insert your receipt" should be added, now it is plain text in the previous line. | C.9.1.3.1                             | A                    | FIXED                    |
| 4         | *Modal popup: close button "X" doesn't own an accessible name: --> assign aria-label="close" to the button element                                                                                              | C.9.4.1.2                             | A                    | FIXED |
| 5         | *Successful report: The page contains allocated text in non semantical  tags. ---> Remove  and add H1                                                                                                          | C.9.1.3.1                             | A                    | NOFIX: Not every text could be semantical                    |
| 6         | *Title " GlobaLeaks" is not adequated to the content. Modify with a coherent title                                                                                                                             | C.9.2.4.2                             | A                    | NOFIX: The title is set dynamically but the used browser seems to contain a bug and do not load it |
| 7         | The focus is not visible on some components (for example: buttons "Previous" and "Next")                                                                                                                       | C.9.2.4.7                             | AA                   | TODO |
| 8         | In the access page the "XXXX XXXX XXXX XXXX" placeholder shoud be deleted or replaced with a semantically appropriate string.                                                                                  | C.9.1.3.1                             | A                    | FIXED |

### Proposed solution

The proposed solutions are exposed in the upper table, the points with an asterisk are those on which we think we can act independently.

**Comment by larrykind** [comment]
Please flag this issue with "Accessibility". Thanks

**Comment by evilaliv3** [comment]
Closing as addressed.
